emergence_lib/src/graphics/mod.rs

```diff
@@ -49,6 +50,7 @@ pub enum GraphicsStage {
 impl Plugin for GraphicsPlugin {
     fn build(&self, app: &mut App) {
         app.add_plugin(bevy_ecs_tilemap::TilemapPlugin)
+            .add_plugin(FrameTimeDiagnosticsPlugin)
```

Comment:
Likely what we will want to do in the future is to put this behind a cargo feature, but for now I am fine with leaving this here as is

But we should make an issue so that we don't forget